sqldelight-compiler/src/test/kotlin/app/cash/sqldelight/core/QueryWrapperTest.kt

```diff
@@ -83,7 +84,7 @@ class QueryWrapperTest {
     )
   }
 
-  @Test fun `queryWrapper has adapter properties`() {
+  @Test @Ignore() fun `queryWrapper has adapter properties`() {
```

Comment:
@AlecStrong from this test it seems like this is intended behavior. Is that the case?

Reply:
I think you can probably slap a query in here that selects from both tables to get the test passing, im not sure what the original issue this was solving is, can prob check the git blame